### PR TITLE
Big swipeable images

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: components/item/AwesomeSwiper.stories.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    template: `
+      <div class="mt-3">
+        <AwesomeSwiper
+          :europeana-identifier="id"
+          :media="media"
+        />
+      </div>
+    `
+  }))
+  .add('Full height images', () => ({
```
Comment (lbiedinger) [inline components/item/AwesomeSwiper.stories.js:34]: let's get rid of this as it's just the same as the other example at the moment.

File: components/item/AwesomeSwiper.vue [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<template>
+  <swiper
+    ref="awesome"
+    class="swiper"
```
Comment (lbiedinger) [inline components/item/AwesomeSwiper.vue:4]: we're not using this class, but might be useful to keep around as we start applying more custom styling ourselves.

File: components/item/AwesomeSwiper.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    },
+    mounted() {
+      // console.log('Current Swiper instance object', this.swiper)
+      // this.swiper.slideTo(3, 1000, false)
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+  /* .swiper-slide {
```
Comment (lbiedinger) [inline components/item/AwesomeSwiper.vue:86]: let's get rid of the unused css.

File: pages/item/_.vue [hunk 4]
```diff
@@ -351,6 +351,8 @@
           this.selectedMedia.about = msg.data.id;
         }
       });
+      console.log(this.media);
```
Comment (lbiedinger) [inline pages/item/_.vue:354]: this can go.

File: components/item/AwesomeSwiper.stories.js (hunk 1 above)
Comment (alansutherland) [reply components/item/AwesomeSwiper.stories.js:34]: ok

File: components/item/AwesomeSwiper.vue (hunk 3 above)
Comment (alansutherland) [reply components/item/AwesomeSwiper.vue:86]: sure, this was on the example but I didn't see the purpose of it so had commented out, will delete

File: pages/item/_.vue (hunk 4 above)
Comment (alansutherland) [reply pages/item/_.vue:354]: ok

**Pull request merged**